mmdet/models/backbones/resnet.py

```diff
@@ -359,6 +296,8 @@ def __init__(self,
                  dilations=(1, 1, 1, 1),
                  out_indices=(0, 1, 2, 3),
                  style='pytorch',
+                 deep_stem=False,
```

Comment:
Use an independent stem module since there may be other forms than the `shallow` and `deep` ones here.